bundles/org.openhab.transform.basicprofiles/src/main/java/org/openhab/transform/basicprofiles/internal/config/InactivityProfileConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.transform.basicprofiles.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.transform.basicprofiles.internal.profiles.InactivityProfile;
+
+/**
+ * Configuration class for {@link InactivityProfile}.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class InactivityProfileConfig {
+    public String timeout = "";
+    public @Nullable Boolean inverted; // (optional) boolean value
```

Comment:
Wouldn't this be simpler and according to the readme this is default.
```suggestion
    public boolean inverted = false;
```